### PR TITLE
feat: implemented auth for login and signup page

### PR DESCRIPTION
**Pull request by therajatg**

### CODE REVIEW

File: src/frontend/pages/signup/signup.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.ecom-placing-input-container {
+  /* height: 100vh; */
```
Comment (Mohitkumar33) [inline src/frontend/pages/signup/signup.css]: remove commented code

**Review by Mohitkumar33 — APPROVED** [review]

File: src/frontend/pages/signup/signup.css (hunk 1 above)
Comment (therajatg) [reply src/frontend/pages/signup/signup.css]: ok

File: src/frontend/pages/signup/signup.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    e.preventDefault();
+    console.log(e);
+    try {
+      const response = await axios.post("/api/auth/signup", {
+        firstName: user.firstName,
+        lastName: user.lastName,
+        email: user.email,
+        password: user.password,
+      });
+      console.log(user.firstName);
```
Comment (sadathshariff) [inline src/frontend/pages/signup/signup.js:22]: Remove Console logs

File: src/frontend/pages/signup/signup.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+  return (
+    <div className="center">
+      <form
+        className="input-container ecom-placing-input-container"
+        onSubmit={formSignupHandler}
+      >
+        <h2 className="ecom-center-inside-flex">CREATE ACCOUNT</h2>
+        <div className="input-name">
+          <label for="firstName">FIRST NAME:</label>
```
Comment (sadathshariff) [inline src/frontend/pages/signup/signup.js:43]: use htmlFor here instead for as react ignores that

**Review by sadathshariff — COMMENTED** [review]
The code looks Clean suggested some points have a look

File: src/frontend/pages/signup/signup.js (hunk 3 above)
Comment (therajatg) [reply src/frontend/pages/signup/signup.js:43]: did not knew this. thanks will change.

File: src/frontend/pages/signup/signup.js (hunk 2 above)
Comment (therajatg) [reply src/frontend/pages/signup/signup.js:22]: will do

**Pull request merged**